soda/core/soda/execution/anomaly_metric_check.py

```diff
@@ -23,59 +28,62 @@ def __init__(
             column=column,
         )
 
+        self.skip_anomaly_check = False
         metric_check_cfg: MetricCheckCfg = self.check_cfg
+        if not metric_check_cfg.fail_threshold_cfg or not metric_check_cfg.warn_threshold_cfg:
```

Comment:
Is this intentional? Because it will shortcut if the `fail_threshold_cfg` is `None` and will skip even if `warn_threshold_cfg` is set.

Reply:
Maybe you meant `if not metric_check_cfg.fail_threshold_cfg and not metric_check_cfg.warn_threshold_cfg:` ?

Reply:
It was intentional, because when `default` is passed this actually gets stored as the `fail_threshold_cfg`, while the `warn_threshold_cfg` is optional so at this point and given the fact that I'm requiring users to use `default` this would work.

Unless you have another suggestion. I do feel, however, that an `and` approach would not be suitable since `warn` can legally be `None`

Reply:
okay - I wasn't sure what default was interpreted to be, in that case, It is fine.